shared/src/test/scala/mouse/FEitherSyntaxTest.scala

```diff
@@ -27,8 +27,8 @@ import cats.syntax.either._
 import scala.util.{Failure, Success, Try}
 
 class FEitherSyntaxTest extends MouseSuite {
-  private val rightValue = List(42.asRight[String])
-  private val leftValue = List("42".asLeft[Int])
+  private val rightValue: Seq[Either[String, Int]] = List(42.asRight[String])
+  private val leftValue: Seq[Either[String, Int]] = List("42".asLeft[Int])
```

Comment:
In the added type ascriptions, there probably should be a `List` to get CI fixed.

Reply:
Reverted them as previous to avoid useless changes, thanks 👍